test/dynamo/test_subclasses.py

```diff
@@ -982,6 +971,38 @@ def fn(x):
             self.assertEqual(res_exp, res_act)
             self.assertEqual(x0, x1)
 
+    def test_subclass_override_shape_and_to(self):
```

Comment:
What if `MySubclass` has a `__torch_function__`? Can we test that we don't inoke that override?

Reply:
Good call lemme add a test, should be handled by this check: https://github.com/pytorch/pytorch/pull/149484/files#diff-b9390381059781e43ca874273a63ac3525d990c9c4d6bb048ef621bbf2a768a2R642-R644

Reply:
Added
- `test_subclass_dont_invoke_torch_function_on_overriden_method`
- `test_subclass_dont_invoke_torch_function_on_overriden_attr`